sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample4_ReadRevisionHistory.cs

```diff
@@ -14,8 +14,8 @@ public partial class ConfigurationSamples
     {
         [Test]
         /*
-         * This sample demonstrates how to read the revision history of one or
-         * more Configuration Settings in a Configuration Store. To do this, we
+         * This sample demonstrates how to read the revision history of a
```

Comment:
Is it ok to have `a` on one line and `Configuration Settings` on another?

Reply:
I think it's ok, just an artifact of text wrapping.  I was trying to keep the header comment at 80 chars wide.